libtransact/build.rs

```diff
@@ -44,6 +44,8 @@ fn main() {
                 .unwrap(),
             proto_path.join("merkle.proto").to_str().unwrap(),
             proto_path.join("command.proto").to_str().unwrap(),
+            #[cfg(feature = "contract-context")]
```

Comment:
This should probably be #[cfg(feature = "key-value-state")] and contract-context-key-value should depend upon it.

Reply:
Features were re-organized 👍